modules/core/src/v2/wallet.ts

```diff
@@ -1359,8 +1366,27 @@ export class Wallet {
     // get keychains for address verification
     const keychains = await Promise.all(this._wallet.keys.map((k) => this.baseCoin.keychains().get({ id: k, reqId })));
     const rootAddress = _.get(this._wallet, 'receiveAddress.address');
+    const addressDerivationKeypair = _.get(keychains[KeyIndices.USER], 'addressDerivationKeypair');
+    const lastChainIndex = _.get(this.coinSpecific(), `lastChainIndex.${chain || 0}`, -1);
```

Comment:
lets default this to `0` instead of `-1` since Address created with Wallet will have index 0.